scripts/zkai_attention_kv_d16_fused_softmax_table_native_gate.py

```
@@ -0,0 +1,755 @@
+    if envelope.get("statement_version") != FUSED_STATEMENT_VERSION:
+        raise AttentionKvD16FusedSoftmaxTableGateError("fused statement version drift")
+    if envelope.get("semantic_scope") != FUSED_SEMANTIC_SCOPE:
+        raise AttentionKvD16FusedSoftmaxTableGateError("fused semantic scope drift")
+    if envelope.get("decision") != DECISION:
+        raise AttentionKvD16FusedSoftmaxTableGateError("fused decision drift")
+    if envelope.get("target_id") != FUSED_TARGET_ID:
+        raise AttentionKvD16FusedSoftmaxTableGateError("fused target id drift")
+    if envelope.get("verifier_domain") != FUSED_VERIFIER_DOMAIN:
+        raise AttentionKvD16FusedSoftmaxTableGateError("fused verifier domain drift")
+    if envelope.get("source_input") != source_input:
+        raise AttentionKvD16FusedSoftmaxTableGateError("fused source input split-brain drift")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)